### ISSUE TITLE
release sgfs of katagotraining.org

### ISSUE THREAD

**Issue by StefanDing** [lightvector/KataGo#460]
hello,the sgfs of older nets has been released on https://d3dndmfyhecmj0.cloudfront.net/
but now the newer sgfs was updated on https://katagotraining.org/
can you release the sgfs of the newer nets?

**Comment by lightvector** [comment]
Yep! It's on the todo list. They will be made available to download in bulk eventually, but it may be many months before that happens, since at the moment there are many other things to do that are slightly higher in priority, or slightly less work to accomplish first, or slightly more fun to work on - which makes them feel like a lot less work, and therefore a lot easier to do, given that this is a weekend hobby project. We'll get there!

If you want it to happen faster, help find volunteers willing to implement it, or to help implement or test other things that need to be worked on too, or help implement it yourself. :) For reference, the website code and infrastructure is pretty much open source at https://github.com/katago/katago-server .

If someone does have some hours to spend each week on coding or research or testing and is looking for a cool project to work on regularly and occupy them, let me know here or on [discord](https://discord.gg/3jfxmrSqgC) and I can talk about some of the things that need some work, or some of the big open directions, or totally new offshoot ideas that might indeed make cool projects if they existed.

**Comment by StefanDing** [comment]
waiting for the newer sgfs...
